client/landing/stepper/declarative-flow/internals/steps-repository/goals/style.scss

```diff
@@ -115,7 +115,7 @@
 		width: 160px;
 	}
 
-	.select-goals__link.is-link { // extra specificity to override default link styles
+	.components-button.select-goals__link.is-link { // extra specificity to override default link styles
```

Comment:
This is giving me pause. There were recent conversations about keeping the components as vanilla as possible, even if it came at the cost of a 1:1 with designs.

Reply:
Yes, I suspect we should be using an onboarding theme that sets text-link color to what we want it to be. There is an onboarding stylesheet but we don't consistently import it and it doesn't apply the styles we want here either.

Links in onboarding all (most) seem to be black text, are these all setting their own overrides?